cli_reference/osdk/cli-osdk-install.adoc

```diff
@@ -8,6 +8,9 @@ toc::[]
 
 The Operator SDK provides a command-line interface (CLI) tool that Operator developers can use to build, test, and deploy an Operator. You can install the Operator SDK CLI on your workstation so that you are prepared to start authoring your own Operators.
 
+:FeatureName: The Operator SDK, including the related Operators,
```

Comment:
Does "related Operators" mean the Helm and Ansible operator base images? If so, I think I'd call that out specifically. I could imagine someone reading this and thinking "OMG! All operators using SDK are deprecated!?"

```suggestion
:FeatureName: The Operator SDK, including the related Helm and Ansible Operator base images,
```

Reply:
I am working on a rewrite of this based on Kevin's feedback. I should have the change pushed shortly.

Reply:
The update is going to say something like:
```suggestion
:FeatureName: The Operator SDK, including the related scaffolding tools and base images for Operator projects,
```

I am also going to make a customized snippet since the OSDK ships with OCP, and is not included in the payload.

Reply:
@joelanford https://github.com/openshift/openshift-docs/pull/77128/commits/72ce2972fd8111841b63e21709eba31a9ad3e68d introduces a more detailed deprecation notice. It details all of the scaffolding tools and base images we currently support. This message is the same as what I added to the [4.16 release notes](https://github.com/openshift/openshift-docs/pull/77128/commits/72ce2972fd8111841b63e21709eba31a9ad3e68d).

If that seems like too much information, https://github.com/openshift/openshift-docs/pull/77128/commits/5cc7f0b54d26809132cacb3dbf3b24bae2ab5eb2 has a shorter deprecation message. I lean towards being more explicit, even if it makes for a rather large admonition.